core/src/main/java/org/elasticsearch/tasks/CancellableTask.java

```diff
@@ -65,4 +66,10 @@ public boolean isCancelled() {
     public String getReasonCancelled() {
         return reason.get();
     }
+
+    /**
+     * Called when the task in cancelled so that it can take any actions that it has to take.
```

Comment:
in -> is

Might be good to specify also whether it's called *before* cancellation, or *after*